releasenotes/notes/depends-on-attrs-19687d5b7a597744.yaml

```
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    The `attrs` library has been removed from `vendor` and the system-wide version is being used.
```

```suggestion
    The `attrs` library has been unvendored and is now required as a normal Python dependency with a minimum version requirement of 19.2.0.
```
unvendored might raise a spellcheck error though, but something like this would make more sense to an outsider reading our release notes 🙂